### DISCUSSION TITLE
How does lazy loading improve page performance?

### DISCUSSION THREAD

**Discussion by Sbiswas001** [Sbiswas001/Student-Portfolio#29] [Q&A] [answered]
Lazy loading is a technique that defers loading of non-critical resources at page load time. Instead, these non-critical resources are loaded at the moment of need.

**Comment by itsyourap** [comment, marked as answer]
> How does lazy loading improve page performance?

- By loading only the resources that are currently needed, lazy loading decreases the amount of data that needs to be downloaded. This is particularly beneficial for users with slow or limited internet connections, as it minimizes unnecessary data transfer.
- Since the critical content loads faster, users perceive the page as being more responsive and quicker.
- By loading only the resources that are necessary at any given time, lazy loading can reduce the number of requests sent to the server. This can lead to decreased server load and improved performance, especially for high-traffic websites.
- It prevents the browser from being overwhelmed by a large number of simultaneous requests and can improve overall stability and performance.
